.travis.yml

```diff
@@ -18,3 +20,5 @@ services:
   - docker
 before_install:
   - (test $RUN_ON_LIVE_SERVER = 1 && ./.travis/start_hawkular_services.sh -d && ./.travis/wait_for_services.rb) || echo "Skipping live server"
+after_success:
+  - ./.travis.docs.rb
```

Comment:
I was looking into it but there were other issues, like very strong assumptions on what you want to do, which was basically uploading a file to somewhere.

I workaround it with a simple hack:
https://github.com/hawkular/hawkular-client-ruby/pull/198/files#diff-515e88bb27e03f935a08a01e14f1d448R17

I.e. it's being run only for the travis job that ends with '.4'